tests/playwright-test/reporter-blob.spec.ts

```diff
@@ -947,6 +947,59 @@ result.stderr: stderr text
 `);
 });
 
+test('encode inline attachments', async ({ runInlineTest, mergeReports }) => {
+  const reportDir = test.info().outputPath('blob-report');
+  const files = {
+    'echo-reporter.js': `
+      import fs from 'fs';
+
+      class EchoReporter {
+        onTestEnd(test, result) {
+          const attachmentBodies = result.attachments.map(a => a.body?.toString('base64'));
```

Comment:
It would be easier to check expectations if this were `toString('utf-8')` instead.